fxapom/pages/page.py

```diff
@@ -34,7 +33,7 @@ def is_element_present(self, *locator):
             return False
         finally:
             # set the implicit wait back
-            self.selenium.implicitly_wait(self.testsetup.default_implicit_wait)
+            self.selenium.implicitly_wait(self.timeout)
```

Comment:
We should avoid setting an implicit wait, unless we set an initial implicit wait.